articles/gitops-mode-software.md

```diff
@@ -0,0 +1,33 @@
+# Manage software using GitOps
+
+This guide is a walkthrough on how to manage software using GitOps. In Fleet, there are three types of software: [Fleet-maintained apps](#fleet-maintained-apps), [App Store (VPP) apps](#app-store-vpp-apps), and [custom packages](#custom-packages).
+
+When using Gitops to manage Fleet, you can optionally put the UI in [GitOps mode](https://fleetdm.com/guides/gitops-mode). This prevents you from making changes in the UI that would be overridden by GitOps workflows.
+
+## Fleet-maintained apps
+
+Currently, managing [Fleet-maintained apps](https://fleetdm.com/guides/fleet-maintained-apps) is only supported using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (Gitops support coming soon. Currently, Gitops won't delete/modify Fleet-maintained apps configured in the UI).
+
+## App Store (VPP) apps
+
+To configure App Store (VPP) apps, via GitOps, please see the [Software section](https://github.com/fleetdm/fleet/blob/main/docs/Configuration/yaml-files.md#software) of Fleet's best practice [GitOps documentation](https://github.com/fleetdm/fleet/blob/main/docs/Configuration/yaml-files.md#gitops). Note that VPP apps must first be added to Apple Business Manager.
```

Comment:
```suggestion
To configure App Store (VPP) apps, via GitOps, please see the [`app_store_apps`](https://github.com/fleetdm/fleet/blob/main/docs/Configuration/yaml-files.md#app_store_apps) of Fleet's best practice [GitOps documentation](https://github.com/fleetdm/fleet/blob/main/docs/Configuration/yaml-files.md#gitops). Note that VPP apps must first be added to [Apple Business Manager](https://business.apple.com).
```

I think we can link directly to the App Store apps section: https://fleetdm.com/docs/configuration/yaml-files#app-store-apps

Also a good opportunity to link directly to the Apple Business Manager login page.